### DISCUSSION TITLE
advanced options

### DISCUSSION THREAD

**Discussion by Cass0182** [pear-devs/pear-desktop#206] [Q&A]
im fairly new to coding, but is there a reason im not able to access the advanced options in the app? or is there something in need to do first?

**Comment by Araxeus** [comment]
Advanced settings button opens a text file with your external text editor
It is saved at `C:\Users\%USERNAME%\AppData\Roaming\YouTube Music\config.json`
(on windows)

But you will soon hopefully almost never need to access that file
See #207 , #243

If you have any other settings you would like to add, hit me up 🙂

**Reply by Cass0182** [reply]
Oh, I was just asking cause I was trying to find a way to integrate it with my stream deck that im using but, it only works on the original YTMusic app, the one that plays ads. I was trying to find a way to make it work with the ad free version of the application 😅

**Reply by Araxeus** [reply]
Try disabling the `shortcuts` plugin and using
[System Actions / Multimedia](https://help.elgato.com/hc/en-us/articles/360028234471-Elgato-Stream-Deck-System-Actions) on your stream deck

(If you disable `shortcuts` on windows/linux, you will get native media support - see #204 )
